lms/djangoapps/support/static/support/js/views/enrollment_modal.js

```diff
@@ -55,6 +55,18 @@
                         // Success callback
                         _.bind(function() {
                             this.hide();
+                            /*
+                            * `/certificates/regenerate` makes the certificate status to unavailable (if it exists)
```

Comment:
@mushtaqak , I think we should include something on the front end to let support know what's happening. 
![unspecified-5](https://cloud.githubusercontent.com/assets/2748698/17627646/fe012a40-607f-11e6-8654-a582af51d796.png)

Like maybe make a checkbox that says "regenerate this user's certificate upon mode change (if we've attempted to generate a cert for this student)" and submit this ajax call only if that checkbox is checked?